microsoft/testsuites/network/stress.py

```diff
@@ -0,0 +1,97 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+from typing import Any, cast
+
+from assertpy import assert_that
+
+from lisa import (
+    Environment,
+    Logger,
+    RemoteNode,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+    node_requirement,
+    schema,
+)
+from lisa.features import Sriov as SriovNetwork
```

Comment:
What's the reason to rename it? It decreases the readability of the code. If someone is familiar with the code, he knows the Sriov well, but don't know SriovNetwork, he will be confused. If the SriovNetwork is a good name, we should rename it in the source.